core/server/common/src/main/webapp/browse-body.jsp

```diff
@@ -17,35 +17,17 @@
 
 <jsp:include page="header-scripts.jsp" />
 
-<script>
-  var currentDir = "<%= request.getAttribute("currentPath").toString() %>";
-  function getInputPath() {
-    var path = $.trim($("#pathInput").val());
-    return path;
-  }
-
-  function changeDir() {
-    var path = getInputPath();
-    path = encodeURI(path);
-    window.location.href = "./browse?path=" + path;
-  }
+<link rel="stylesheet" href="css/ui-grid.css" type="text/css">
```

Comment:
What are the versions of angular and ui-grid?
Can we use some version control tool like npm to manage the dependency?